tests/arp/test_arpall.py

```diff
@@ -51,7 +52,7 @@ def common_setup_teardown(duthost, ptfhost):
         yield duthost, ptfhost, int_facts, intf1, intf2, intf1_indice, intf2_indice
     finally:
         # Recover DUT interface IP address
-        duthost.shell("sudo config reload -y &>/dev/null", executable="/bin/bash")
+        config_reload(duthost, config_source='minigraph', wait=120)
```

Comment:
Does it need to be minigraph? The previous code translate to config_source='config_db'.

Reply:
I am not quite sure which way is recommended. I saw another test is using minigraph to restore the configuration. I guess it might be due to our DUT were setup with minigraph?
Since we are not using config_db.json to deploy our testbed, I think minigraph is the source of true.

Reply:
True. We do have minigraph, but minigraph is converted to config_db.json before being used. For us, reload config is the lighter weight recover method. load minigraph is heavier weight recover method. I believe in most cases, reload config (config_db) is sufficient.

And for community members that don't use minigraph, they won't be able to fall back to this method. therefore. config_db recover is also more generic.

Reply:
Make scenes! I updated the PR